examples/gtest/CMakeLists.txt

```
@@ -0,0 +1,43 @@
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
```

This silences the CMake warning about the missing `project()` command. I'll also [add it](https://github.com/TheLartians/CPM/pull/83/commits/4afc4f09d59857bda8c40d9e19dff6eba844d812) to the other examples with for the next version.

```suggestion

project(CPMExampleGtest)

```